tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-cpu.yml

```diff
@@ -0,0 +1,102 @@
+trigger: none
+
+stages:
+- stage: Python_Packaging
+
+  jobs:
+    - job: Linux_Training_CPU_Wheels
+      timeoutInMinutes: 60
+      workspace:
+        clean: all
+      pool: Onnxruntime-Linux-GPU-NV6
```

Comment:
If it is a cpu docker image, do you need NV6?

Reply:
changed to Linux-CPU

Reply:
changed to Linux-CPU